ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -713,6 +735,18 @@ impl AotIRDisplay for Instruction {
                     .collect::<Vec<_>>();
                 ret.push_str(&format!("phi {}", args.join(", ")));
             }
+            Self::IndirectCall {
+                fty_idx: _,
+                callop,
+                args,
+            } => {
+                let args_s = args
+                    .iter()
+                    .map(|a| a.to_string(m))
+                    .collect::<Vec<_>>()
+                    .join(", ");
+                ret.push_str(&format!("call {}({})", callop.to_string(m), args_s,));
```

Comment:
Did `cargo fmt` allow that trailing comma?

Reply:
Seems so.

Reply:
Fair enough. Remove the comma if you want.

Reply:
21b461be499ecfc90c506161e0db230d24ef9699